.circleci/config.yml

```diff
@@ -82,7 +82,7 @@ jobs:
                 path: ./test-results
             - store_artifacts:
                 path: ./test-results
-          no_output_timeout: 120m
+          no_output_timeout: 140m
```

Comment:
Do we need to have this match the test timeout? Every time a test case runs, its description is output, and I don't think we have a test that can take longer than `120m`.

Reply:
yeah, I wasn't sure how this one works. I think I had to increase it at some point but maybe it wasn't this one that I needed to change. Let's see what happens tonight when the IT run and we can decide on Monday.